sklearn/metrics/classification.py

```diff
@@ -667,7 +667,8 @@ def f1_score(y_true, y_pred, labels=None, pos_label=1, average='binary',
 
     References
     ----------
-    .. [1] `Wikipedia entry for the F1-score <https://en.wikipedia.org/wiki/F1_score>`_
+    .. [1] `Wikipedia entry for the F1-score
```

Comment:
I think it would. It's not different from docstrings of `confusion_matrix()` (line 215) and `cohen_kappa_score()` (line 324). I didn't change them and it seems that they are correctly rendered (see [confusion_matrix](http://scikit-learn.org/dev/modules/generated/sklearn.metrics.confusion_matrix.html#sklearn.metrics.confusion_matrix) and [cohen_kappa_score](http://scikit-learn.org/dev/modules/generated/sklearn.metrics.cohen_kappa_score.html#sklearn.metrics.cohen_kappa_score))